guides/manual-installation.md

```diff
@@ -70,6 +70,26 @@ Problem: nothing provides 'group(prometheus)' needed by the to be installed gola
  Solution 2: break golang-github-prometheus-prometheus-2.37.6-150100.4.17.1.x86_64 by ignoring some of its dependencies
 ```
 
+Change prometheus configuration by replacing the configuration at `/etc/prometheus/prometheus.yml` with:
+
+```bash
+global:
+  scrape_interval: 30s
+  evaluation_interval: 10s
+
+scrape_configs:
+  - job_name: "http_sd_hosts"
+    honor_timestamps: true
+    scrape_interval: 30s
+    scrape_timeout: 30s
+    scheme: http
+    follow_redirects: true
+    http_sd_configs:
+      - follow_redirects: true
+        refresh_interval: 1m
+        url: http://localhost:4000/api/prometheus/targets
```

Comment:
We should point out that this `localhost:4000` is where the web container is running.